gio/src/unix_mount_entry.rs

```diff
@@ -61,7 +61,7 @@ impl UnixMountEntry {
 
     #[doc(alias = "g_unix_mount_get_device_path")]
     #[doc(alias = "get_device_path")]
-    pub fn device_path(&self) -> Option<std::path::PathBuf> {
+    pub fn device_path(&self) -> std::path::PathBuf {
```

Comment:
Why are all these manually written? For `UnixMountPoint` (which is more or less the same API!) almost all these functions are auto-generated.

Can you check? That would automatically do the right thing with regards to nullability also.

Reply:
Does the gir matching work via function names? The functions are called `g_unix_mount_point_*()` (correct) and `g_unix_mount_*()` (incorrect). I didn't found a way to manually add functions to a object in gir or to assign functions to an object. When generated naively the `g_unix_mount_*()` end up in `functions`.

Reply:
Uh, ok. So that's the reason why :( Not sure why that happens, it looks correct to me in the gir. Can you create an issue about that, and then let's get this one in?